benchmark.py

```diff
@@ -53,7 +53,7 @@ class Article(Model):
         __tablename__ = "article"
         __versioned__ = copy(options)
 
-        id = sa.Column(sa.Integer, autoincrement=True, primary_key=True)
+        id = sa.Column(sa.Integer, sa.schema.Sequence("article_id_seq"), autoincrement=True, primary_key=True)
```

Comment:
Can we use identity instead ?

Reply:
Updated to Identity for all testcases there were certain cases where Identity wasn't working have seperated them out in commits and explained the issue faced.

Reply:
It seems Identity column is causing issues with MSSQL DB where it requires `IDENTITY_INSERT` to be ON